azext_edge/edge/providers/orchestration/resources/schema_registries.py

```diff
@@ -66,7 +67,11 @@ def create(
         custom_role_id: Optional[str] = None,
         **kwargs,
     ) -> dict:
+        from ..rp_namespace import register_providers
         with console.status("Working...") as c:
+            # Register providers - may as well do all for AIO
```

Comment:
Please only register the Schema Registry RP, the way you have it here might lead to unintended consequences.

Reply:
ok - keeping the adr register in init for now